src/components/FieldTypes/MultipleSelect.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+import { defineProps } from "vue";
+import { ref, computed } from "vue";
+
+const props = defineProps({
+  missingFieldName: String,
+  inputOptions: Array,
+  description: String,
+  modelValue: String,
```

Comment:
Should be `modelValue: Array`

Reply:
This makes sense. I have made the change.